src/NetworkEntities.js

```diff
@@ -35,15 +35,15 @@ class NetworkEntities {
     var hasPosition = componentData.hasOwnProperty('position');
     if (hasPosition) {
       var position = componentData.position;
-      entity.setAttribute('position', position);
+      entity.setAttribute('position', `${position.x} ${position.y} ${position.z}`);
```

Comment:
I thought A-Frame supported passing in objects rather than needing to serialize to a string here? Why the change?

Reply:
It does, but only after the `a-entity` Custom HTMLElement is done overwriting `setAttribute`.

I've created an example project to help myself and others learn about the Custom HTMLElement lifecycle.

https://aframe-custom-element-lifecycle.glitch.me/

Maybe we can make everything work asynchronously to deal with `setAttribute` not being overwritten until the next tick?